src/services/contentWarningService.ts

```
@@ -0,0 +1,145 @@
+
+export interface ContentWarningResult {
+  hasWarning: boolean;
+  categories: WarningCategory[];
+  matchedTerms: string[];
+}
+
+// ── Sensitive Topic Dictionary ───────────────────────────────
+
+const SENSITIVE_TERMS: Record<WarningCategory, string[]> = {
+  Violence: [
+    "violence", "violent", "assault", "abuse", "abusive", "trauma",
```

ESLint

[error] Replace `·"violent",·"assault",·"abuse",·"abusive",` with `⏎····"violent",⏎····"assault",⏎····"abuse",⏎····"abusive",⏎···`

(prettier/prettier)